float32 linear_x
float32 angular_z
bool reset